charts/fluent-operator/values.yaml

```diff
@@ -63,7 +63,24 @@ fluentbit:
       enable: false
       host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
       port: 9200
-      logstashPrefix: ks-logstash-log
+#      path: ""
+#      bufferSize: "4KB"
+#      index: "fluent-bit"
+#      httpUser:
+#      httpPassword:
+#      logstashFormat: true
+#      logstashPrefix: ks-logstash-log
```

Comment:
`#      logstashPrefix: ks-logstash-log` can be removed here

Reply:
Done.